util/perl/TLSProxy/Proxy.pm

```diff
@@ -450,23 +454,23 @@ sub supports_IPv6
     return $have_IPv6;
 }
 
-#Read/write accessors
-sub proxy_addr
-{
-    my $self = shift;
-    if (@_) {
-        $self->{proxy_addr} = shift;
-    }
-    return $self->{proxy_addr};
-}
-sub proxy_port
-{
-    my $self = shift;
-    if (@_) {
-        $self->{proxy_port} = shift;
-    }
-    return $self->{proxy_port};
-}
+##Read/write accessors
+#sub proxy_addr
+#{
+#    my $self = shift;
+#    if (@_) {
+#        $self->{proxy_addr} = shift;
+#    }
+#    return $self->{proxy_addr};
+#}
+#sub proxy_port
+#{
+#    my $self = shift;
+#    if (@_) {
+#        $self->{proxy_port} = shift;
+#    }
+#    return $self->{proxy_port};
+#}
```

Comment:
Why comment this out? What is your motivation for wanting to remove them? Either we should keep this code and continue to call it - or we should delete it entirely.

Reply:
The reason why I commented it out is that it's unusable in its current form.  These functions *should* close the proxy socket and re-create it with the new data, but considering they aren't called anywhere, I'm unsure it's worth the effort.  So I commented them out for now.

You thoughts, please?  They matter.

Reply:
Well they weren't called as setters, but they were called as getters (you changed it to not do that as part of this PR). But as there is only one call site per function I really don't mind too much. I'm happy for them to go - but only if they are deleted properly (not just commented out). The other alternative is to simplify them and just make them simple getters.

Reply:
Ok, made them accessor only